src/tests/unit/tests/electron/views/device-connect-view/components/android-setup/__snapshots__/android-setup-step-container.test.tsx.snap

```diff
@@ -13,20 +13,15 @@ exports[`AndroidSetupStepContainer passes common props and deps through 1`] = `
     deps={
       Object {
         "LinkComponent": [Function],
-        "androidSetupActionCreator": null,
         "androidSetupStepComponentProvider": Object {
           "detect-adb": [Function],
           "prompt-locate-adb": [Function],
         },
-        "closeApp": null,
```

Comment:
Glad to see these going away--it was not a good code smell to have to change a bunch of downstream test classes when adding a field to the properties